tools/pretty_printer.py

```diff
@@ -55,39 +50,28 @@ def trade_pretty_printer(trade):
             except KeyError:
                 order_type_name = trade.order_type.__class__.__name__
 
-        return "{0}: {1} {2} at {3} {4} on {5}: {6} ".format(
-            order_type_name,
-            PrettyPrinter.get_min_string_from_number(trade.quantity),
-            currency,
-            PrettyPrinter.get_min_string_from_number(trade.price),
-            market,
-            trade.exchange.get_name(),
-            convert_timestamp_to_datetime(trade.filled_time))
+        return f"{order_type_name}: {PrettyPrinter.get_min_string_from_number(trade.quantity)} {currency} " \
+            f"at {PrettyPrinter.get_min_string_from_number(trade.price)} {market} on {trade.exchange.get_name()}: " \
+            f"{convert_timestamp_to_datetime(trade.filled_time)} "
 
     @staticmethod
     def cryptocurrency_alert(crypto_currency, symbol, result, final_eval):
-        return "OctoBot ALERT : #{0}\n Symbol : #{1}\n Result : {2}\n Evaluation : {3}".format(
-            crypto_currency,
-            symbol.replace("/", ""),
-            str(result).split(".")[1],
-            final_eval)
+        return f"OctoBot ALERT : #{crypto_currency}\n Symbol : #{symbol.replace('/', '')}\n " \
+            f"Result : {str(result).split('.')[1]}\n Evaluation : {final_eval}"
 
     @staticmethod
     def global_portfolio_pretty_print(global_portfolio, separator="\n"):
-        result = ["{0} ({1}) {2}".format(
-            PrettyPrinter.get_min_string_from_number(amounts[Portfolio.TOTAL]),
-            PrettyPrinter.get_min_string_from_number(amounts[Portfolio.AVAILABLE]),
-            currency)
-            for currency, amounts in global_portfolio.items() if amounts[Portfolio.TOTAL] > 0]
+        result = [f"{PrettyPrinter.get_min_string_from_number(amounts[Portfolio.TOTAL])} " \
```

Comment:
I think that a [_Multi-line string syntax_](https://www.smallsurething.com/multi-line-strings-in-python/) will be better here. (with `"""` )